explorer/graphql/app/metrics.go

```diff
@@ -0,0 +1,72 @@
+package app
+
+import (
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+// instrumentHandler wraps an http.Handler and provides Prometheus metrics for it.
+func instrumentHandler(h http.Handler, name string) http.Handler {
+	inFlight := promauto.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
lets follow our existing pattern, see any `metrics.go` file for how we register metrics.

Reply:
we register metrics as global variables...